review_practice/react-app/src/components/GlobalStyle.js

```diff
@@ -0,0 +1,27 @@
+import { createGlobalStyle } from 'styled-components';
+
+export default createGlobalStyle`
+    * {
+        --primary-basic-color: rgb(255, 90, 95);
+        --primary-light-color: rgb(255, 150, 154);
+
+        margin: 0;
+        padding: 0;
+        border: 0;
+        font-size: 100%;
+        font: inherit;
+        box-sizing: boerder-box;
+    }
+
+    html {
+        font-size: 16px;
+    }
+
+    html, body, .root {
+        height: 100%;
+    }
+
+    body {
+        font-family: 'Gugi';
```

Comment:
```suggestion
        font-family: 'Gugi', sans-serif;
```

"Gugi" 폰트가 뭔지 모르겠지만, `font-family` 를 작성한 뒤에는 꼭 산세리프인지, 세리프 폰트인지 명시해줄 필요가 있습니다. 맥 사파리와 같은 브라우저에서는 기본 폰트를 세리프로 사용하기 때문에 명시하지 않으면 "Gugi" 폰트를 지원하지 않는 맥 사파리 환경에서는 세리프 폰트가 나오게 됩니다.